examples/Makefile-nrf52840

```diff
@@ -79,6 +79,10 @@ COMMONCFLAGS                   := \
     -I$(CONFIG_FILE_PATH)         \
     $(NULL)
 
+ifeq ($(CERT_LOG),1)
+COMMONCFLAGS += -DOPENTHREAD_CONFIG_ENABLE_DEFAULT_LOG_OUTPUT=1
```

Comment:
I think you can instead use the platform specific config file `examples/platforms/nrf52840/openthread-core-nrf52840-config.h` and add this (see below) as a define there... This header file is intended for all `OPENTHREAD_CONFIG_<>` options specific to a project/platform.
```
/**
 * @def OPENTHREAD_CONFIG_ENABLE_DEFAULT_LOG_OUTPUT
 *
 * Define to 1 to enable default log output.
 *
 */
#define OPENTHREAD_CONFIG_ENABLE_DEFAULT_LOG_OUTPUT            1
```
(may also need to include this header file in `platform.c`)...